test/pyWrapper/utils.py

```diff
@@ -3,8 +3,13 @@
 import json
 import numpy as np
 import matplotlib.pyplot as plt
+<<<<<<< HEAD
+import pyvista as pv
```

Comment:
There is an error here. An incomplete merge

Reply:
Fixed